codegen/smithy-go-codegen/src/main/java/software/amazon/smithy/go/codegen/integration/Waiters.java

```diff
@@ -511,7 +512,7 @@ private void generateRetryable(
                                     // if modeled error shape
                                     if (errorShape.isPresent()) {
                                         Symbol modeledErrorSymbol = SymbolUtils.createValueSymbolBuilder(
-                                                errorShape.get().getName(), "types"
+                                                errorShape.get().getName(), operationSymbol.getNamespace() + "/types"
                                         ).build();
```

Comment:
Would it be better to get a symbol of `errorShape` and use the built in symbol formatting for that to get its name and import?